modules/scope/variables.tf

```
@@ -0,0 +1,16 @@
+variable "ipam_id" {
+  description = "(Required) The ID of the IPAM for which you're creating this scope."
+  type        = string
+}
+
+variable "description" {
+  description = "(Optional) A description for the scope you're creating."
+  type        = string
+  default     = "My IPAM Scope"
```

Defaults for descriptions can usually just be `null`

```suggestion
  default     = null
```